### PR TITLE
Fix scheduled reports for mysql

### PR DESCRIPTION
**Pull request by mahendra**
@mistercrunch @timifasubaa @cloud26 - A tentative fix to the mysql issue.

I tried it with postgres and it works.
Sqlite db - I get errors saying "db is locked".

@cloud26 - can you try it with mysql please?

### CODE REVIEW

**Comment by mahendra** [comment]
would anyone know why it locks up sqlite?

**Comment by cloud26** [comment]
yes, I tried this way a few days ago. It works with mysql.

I also met problem that db is locked when I tried to update a dashboard email schedule or slice email schedule.

**Review by phoenix24 — APPROVED** [review]

**Comment by mahendra** [comment]
@cloud26 - can you please confirm that the dashboard email schedule editing was not locking after this fix? Once you confirm, I will merge this in. 

Thank you @phoenix24 for the approval.

**Comment by mistercrunch** [comment]
What is the MySQL issue you are referring to?

File: setup.py [hunk 1]
```diff
@@ -94,9 +94,6 @@ def get_git_sha():
         'thrift-sasl>=0.2.1',
         'unicodecsv',
         'unidecode>=0.04.21',
-        'croniter==0.3.25',
```
Comment (mistercrunch) [inline setup.py]: Are these deps not required anymore?

Comment (mahendra) [reply setup.py]: (apologies for the late reply here. I was on vacation and traveling)

These dependencies are still needed, but they are specified in `requirements.txt` already. Did not want to keep it in two places.

**Comment by mahendra** [comment]
> What is the MySQL issue you are referring to?

https://github.com/apache/incubator-superset/pull/5294#issuecomment-443069997 

@mistercrunch this was the issue.

File: setup.py (hunk 1 above)
Comment (mistercrunch) [reply setup.py]: `requirements.txt` is a pip-compiled, "pinned" version of what's in here. We probably want to keep this in here though I think it belong in an `extra_requires` sub-category, but that's outside the scope of this PR. For now let's just keep this in the repo. Mind remove these edits from this PR?

**Review by mistercrunch — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (mahendra) [reply setup.py]: @mistercrunch removed the edits, but did alphabetically sort it. Let me know if you are OK with it and I will merge it in.

**Comment by mahendra** [comment]
@mistercrunch approve please?

**Review by mistercrunch — APPROVED** [review]

**Pull request merged**